controllers/grid/files/final/form/ManageFinalDraftFilesForm.inc.php

```diff
@@ -36,7 +36,7 @@ function __construct($submissionId) {
 	 * @param $fileStage int SUBMISSION_FILE_...
 	 * @return array a list of all submission files marked as "final".
 	 */
-	function execute($stageSubmissionFiles, $fileStage = null) {
+	function execute($stageSubmissionFiles = null, $fileStage = null, ...$functionArgs) {
```

Comment:
The `...$functionArgs` should be passed into the `parent::execute` call.

Reply:
Fixed